prody/dynamics/clustenm.py

```diff
@@ -967,10 +984,17 @@ def run(self, cutoff=15., n_modes=3, gamma=1., n_confs=50, rmsd=1.0,
         :arg platform: Architecture on which the OpenMM part runs, default is None.
             It can be chosen as 'CUDA', 'OpenCL' or 'CPU'.
             For efficiency, 'CUDA' or 'OpenCL' is recommended.
+            'CPU' is needed for setting threads per simulation.
         :type platform: str
 
         :arg parallel: If it is True (default is False), conformer generation will be parallelized.
+            This can also be set to a number for how many simulations are run in parallel.
```

Comment:
This line is flawed I agree. It should say how many parallel processes are used for pooling the conformer generation and not have anything to do with simulations.